src/main/app/src/Components/Landscape/Overview/Overview.tsx

```diff
@@ -18,19 +18,24 @@ const Overview: React.FC<Props> = ({ setSidebarContent, setPageTitle }) => {
   const [landscapes, setLandscapes] = useState<ILandscape[]>([]);
   const [landscapeLinks, setLandscapeLinks] = useState<ILandscapeLinks | null>();
   const [loadLandscapes, setLoadLandscapes] = useState<boolean>(true);
+  const [landscapesCount, setLandscapesCount] = useState<Number | null>();
+
 
   const getLandscapes = useCallback(async () => {
     if (loadLandscapes) {
       setLandscapeLinks(await get('/api/'));
       if (landscapeLinks) {
+        let count = 0;
```

Comment:
Is the count necessary at all? Why not use size/length?

Reply:
yes, the count is necessary because landscapeLinks._links -data from backend is JSON object instead of JSON array, so I can't use size or length. yet afraid to change from backend as it might affect the main functionality and requires big change.

Reply:
`Object.keys(obj).length` would this work? https://stackoverflow.com/questions/126100/how-to-efficiently-count-the-number-of-keys-properties-of-an-object-in-javascrip